CorsixTH/Src/th_gfx.cpp

```diff
@@ -142,6 +146,47 @@ class Input
     }
 };
 
+//! Convert legacy 8bpp sprite data to recoloured 32bpp data, using special recolour table 0xFF.
+/*!
+@param pPixelData Legacy 8bpp pixels.
+@param iPixelDataLength Number of pixels in the \a pPixelData.
+@return Converted 32bpp pixel data, if succeeded else nullptr is returned. Caller should free the returned memory.
+*/
```

Comment:
Corsix documents non-static function in the header files.
(I don't agree with this policy, but that's another matter.)